draft-ietf-quic-transport.md

```diff
@@ -4872,11 +4873,11 @@ max_ack_delay (0x0b):
 disable_active_migration (0x0c):
 
 : The disable active migration transport parameter is included if the endpoint
-  does not support active connection migration ({{migration}}). Peers of an
-  endpoint that sets this transport parameter MUST NOT send any packets,
-  including probing packets ({{probing}}), from a local address or port other
-  than that used to perform the handshake.  This parameter is a zero-length
-  value.
+  does not support active connection migration ({{migration}}) on the address
+  being used during the handshake.  When a peer sets this transport parameter,
+  the recipient MUST NOT send any packets to the address the peer used to
+  perform the handshake from a local address or port other than that used to
+  perform the handshake.  This parameter is a zero-length value.
```

Comment:
I would reiterate the sentence from above here:
> The disable_active_migration transport parameter does not prohibit connection migration after a client has acted on the preferred_address transport parameter.

I fear that otherwise folks might forget that detail as it is very much not clear from the name of the parameter